app/controllers/users_controller.rb

```diff
@@ -79,7 +79,7 @@ def update
         if password_changed
           redirect_to new_user_session_path
         else
-          render :edit
+          redirect_to edit_user_path
```

Comment:
This doesn't break showing validation errors on the fields? Or I guess we don't do it there at all...